wallet/wallet.go

```diff
@@ -720,7 +719,9 @@ func (w *Wallet) diskWriter() {
 // there are any transactions with outputs to this address in the blockchain or
 // the btcd mempool.
 func (w *Wallet) AddressUsed(addr waddrmgr.ManagedAddress) bool {
-	return addr.Used()
+	// TODO: handle err
```

Comment:
Can you add this?

In fact, just remove this function.  The caller already has the managed address, so they can check this themselves.